### ISSUE TITLE
Feature Request. Learning Schedule options.

### ISSUE THREAD

**Issue by vternal3** [tensorflow/playground#158]
Add learning scheduler options such as the Cycle Triangle among others. 

This link speaks about the variety of learning types
https://medium.com/analytics-vidhya/cyclical-learning-rates-a922a60e8c04

**Comment by emmanuelrouxfr** [comment]
Dear @vternal3, by reading your feature request it seems that you may appreciate the fork we've been working on !

https://coolearning.github.io/playground/

The Coolearning interface would allow you to interactively change the learning rate (and this for each neuron individually if you like) and much more ! Like individual activation functions or regularization for each neuron also !

See you there :)
